bundles/org.openhab.binding.wundergroundupdatereceiver/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,512 @@
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Wind</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="wind-gust-speed-10min">
+		<item-type>Number:Speed</item-type>
+		<label>Gust Speed 10min Average</label>
+		<description>10 minute average gust speed.</description>
+		<category>Wind</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Wind</tag>
+		</tags>
+		<state readOnly="true" pattern="%.2f m/s"/>
+	</channel-type>
+
+	<channel-type id="wind-gust-direction-10min">
+		<item-type>Number:Angle</item-type>
+		<label>Gust Direction 10min Average</label>
+		<description>10 minute average gust direction.</description>
+		<category>Wind</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Wind</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="humidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>Humidity in %.</description>
+		<category>Humidity</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Humidity</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="indoor-humidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Indoor Humidity</label>
+		<description>Indoor humidity in %.</description>
+		<category>Humidity</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Humidity</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="dew-point">
+		<item-type>Number:Temperature</item-type>
+		<label>Dew Point</label>
+		<description>Outdoor dew point.</description>
+		<category>Humidity</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<!-- for extra outdoor sensors use temp2f, temp3f, and so on -->
+	<channel-type id="indoor-temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Indoor Temperature</label>
+		<description>Indoor temperature.</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<!-- for sensors 2,3,4 use soiltemp2f, soiltemp3f, and soiltemp4f -->
+	<channel-type id="soil-temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Soil Temperature</label>
+		<description>Soil temperature.</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="rain">
+		<item-type>Number:Length</item-type>
+		<label>Rain in past hour</label>
+		<description>Rain over the past hour.</description>
+		<category>Rain</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Rain</tag>
+		</tags>
+		<state readOnly="true" pattern="%.2f mm"/>
+	</channel-type>
+
+	<channel-type id="rain-daily">
+		<item-type>Number:Length</item-type>
+		<label>Rain since start of day</label>
```

Comment:
Labels should be short and all words start with a capital. Suggest this one could be called `Daily Rain`.